core/src/protobuf.py

```diff
@@ -237,7 +241,7 @@ def load_message(
                 raise ValueError
             continue
 
-        fname, ftype, fflags = field
+        fname, ftype, fdefault = field
```

Comment:
hmm
Well, it can only contain _a_ flag, singular. And perhaps "flag" is not the right word either, it's more like a placeholder for "what happened to the default value"; neither repeated nor required fields _can_ have a default.

Technically speaking `FLAG_REPEATED` wouldn't need to exist, we could put in `list` as the default value, and add something like `if is_callable(fdefault): fdefault = fdefault()`
But that's a feature that we don't actually need...

Reply:
So maybe just a comment?